README.md

```diff
@@ -1 +1,276 @@
-# capstone-onlie-ticket-
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="" alt="logo" width="140"  height="auto" /> -->
+  <br />
+
+</div>
+
```

Comment:
- [ OPTIONAL ] I will suggest that you remove these from your README to make your repository more professional. You can even see that on line 25, you are asked to remove all comments and Instructions once you're done setting up your README file so kindly ensure that you do this throughout your README file:100: